app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
```

Comment:
I wonder if this check is even necessary. Any thoughts? Is it possible that the super constructor is creating a role for the user?